### DISCUSSION TITLE
Supported dongles/connectors

### DISCUSSION THREAD

**Discussion by parautenbach** [kellerza/sunsynk#87] [General]
Are only the [Solarman dongles ](https://www.solarmanpv.com/products/data-logger/stick-logger/)supported, or will this work with [Sunsynk's own data logger](https://www.sunsynk.org/remote-monitoring) too?

I know the Sunsynk data logger is specifically for remote access via Sunsynk's platform. I'm not interested in that though. I'm simply looking to integrate this with Home Assistant.

**Comment by kababoom** [comment]
This addon uses rs485 -> USB, not the dongle.

**Comment by kellerza** [comment]
The repo's main readme has a couple of examples of USB-RS485 adaptors

**Reply by kellerza** [reply]
https://kellerza.github.io/sunsynk/guide/deployment-options#usb-to-rs485-adaptors
